frontend/components/Gallery/ProjectPreview.js

```diff
@@ -1,19 +1,19 @@
 import {
     client as lensClient,
     mirror,
-    getProfileId,
+    getProfileByAddress,
     createProfile,
-} from '../../api';
+} from '../../clientApi';
 import Link from 'next/link';
 const lensContract = '0x60Ae865ee4C725cd04353b5AAb364553f56ceF82';
 
 export default function ProjectPreview(props) {
     async function mirrorPost() {
         try {
             const addressId = await lensClient.query({
-                query: getProfileId,
+                query: getProfileByAddress,
                 variables: {
-                    owner: [props.address],
+                    owner: props.address,
```

Comment:
Smart move!!! I was trying to find how to make it this way but I was having an error 😁 you rock JP